src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Network</PackageId>
     <Description>Provides management capabilities for Network services.</Description>
     <AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-    <Version>14.1.0-preview</Version>
+    <Version>15.1.0-preview</Version>
```

Comment:
@EvgenyAgafonchikov mark this as 15.0.0
Because after you merge this I am going to cut a GitHub release for 15.0.0 that was released.
This way anytime we need symbols or sources for 15.0.0 they will be available. So don't mark this as 15.1.0